src/index.js

```diff
@@ -0,0 +1,122 @@
+const urlLocation = 'http://127.0.0.1:5000/location';
+const urlWeather = 'http://127.0.0.1:5000/weather';
+
+const windowLoad = () => {
+  const tempUp = document.getElementById('temp-up');
+  const tempDown = document.getElementById('temp-down');
+  const getRealTemp = document.getElementById('realtime');
+  const resetButton = document.getElementById('reset-button');
+
+  let temperature = document.getElementById('display-temp');
+  let cityName = document.getElementById('city-name');
+  let defaultCity = 'Seattle';
+  let selectSky = document.getElementById('sky');
+  let sky = document.getElementById('sky-emoji');
+  let inputCity = document.getElementById('input-city');
+
+  cityName.innerHTML = defaultCity;
+  temperature.innerHTML = getRealtimeTemp(cityName, temperature);
```

Comment:
`getRealtimeTemp` does not return anything. It mutates the HTML itself, we do not need to assign the return value to anything.